docs/components/input-controller/mux.md

```diff
@@ -79,9 +81,9 @@ Click **Create component** and then fill in the attributes for your model:
 
 The following attributes are available for `mux` input controllers:
 
-| Name | Inclusion | Description |
-| ---- | --------- | ----------- |
-| `sources` | *Required* | An array of `name`s of each input controller component that should be combined in the `mux` model. |
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `sources` | array | **Required** | The `name` of each controller you wish to combine. |
```

Comment:
```suggestion
| `sources` | array | **Required** | The `name`s of all controller components you wish to combine. |
```
Or something to make it clearly plural, and I think leaving "component" in helps them know it's the config file names?

Reply:
Good suggestion but I think this should be fine in the more brief form w/ example and "each"

Reply:
Alternative: 
```suggestion
| `sources` | array | **Required** | The `name` of each controller component you wish to combine the input from. |
```